### PR TITLE
Nishithshowri006 patch 1

### PR DESCRIPTION
**Pull request by nishithshowri006**

### CODE REVIEW

File: web_programming/get_imdb_top_250_movies_csv.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import csv
+from typing import Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_imdb_top_250_movies(url: str = "") -> Dict[str, float]:
```
Comment (onlinejudge95) [inline web_programming/get_imdb_top_250_movies_csv.py]: Can we get some `doctests` here

File: web_programming/get_imdb_top_250_movies_csv.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    soup = BeautifulSoup(requests.get(url).text, "html.parser")
+    titles = soup.find_all("td", attrs="titleColumn")
+    ratings = soup.find_all("td", class_="ratingColumn imdbRating")
+    return {
+        title.a.text: float(rating.strong.text)
+        for title, rating in zip(titles, ratings)
+    }
+
+
+def write_movies(filename: str = "IMDb_Top_250_Movies.csv") -> None:
```
Comment (onlinejudge95) [inline web_programming/get_imdb_top_250_movies_csv.py]: `doctests`

File: web_programming/get_imdb_top_250_movies_csv.py (hunk 1 above)
Comment (nishithshowri006) [reply web_programming/get_imdb_top_250_movies_csv.py]: added doc strings

Comment (cclauss) [reply web_programming/get_imdb_top_250_movies_csv.py]: This is network access code.  I am not clear how we would doctest it without resorting to mocking or really slowing down our test cycles, both of which might be overkill.  Note that most of the algorithms in the __ftp__ and __web_programming__ directories do not have doctests and I am OK with that.  Thoughts?

Comment (cclauss) [reply web_programming/get_imdb_top_250_movies_csv.py]: See https://www.python.org/dev/peps/pep-0008/#documentation-strings for conventions on docstrings.  psi/black will change ''' to """ so you might as well do that.

Comment (nishithshowri006) [reply web_programming/get_imdb_top_250_movies_csv.py]: @cclauss I am noob, I have doubt whenever I want to upload a new file should open a new branch and make the pull request?

Comment (cclauss) [reply web_programming/get_imdb_top_250_movies_csv.py]:
```
git checkout master
git pull --rebase upstream master
git push origin master
git branch
git checkout -b my_new_idea
```

Comment (nishithshowri006) [reply web_programming/get_imdb_top_250_movies_csv.py]: thanks

**Pull request closed without merging**